doc/sphinx/tutorial/installation.md

```diff
@@ -0,0 +1,49 @@
+# Installation
+The Mir project provides libraries that aid in the development of a compositor.
+The primary libraries that you will interact with are:
+
+- `libmiral` - short for "Mir Abstraction Layer". Provides a high level interface to interact with and customize Mir compositors.
+- `mir-graphics-drivers-desktop` - provides drivers so Mir can talk with different graphics drivers
```

Comment:
A tutorial is goal oriented, and a guide to doing something meaningful to the reader. We should aim for that...

"In this tutorial you will learn how to install the development packages for Mir so that you can develop your compositor."